apps/linkedin-to-notion/src/popup/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { BaseParagraph } from 'design-system';
+import { useEffect, useState } from 'react';
+
+import { linkedInURLRegex } from '~src/background';
+import { GoBackToLinkedInContent } from '~src/components/GoBackToLinkedIn/GoBackToLinkedIn';
+
+import '~style.css';
+
+export default function Popup() {
+  const [isLinkedIn, setIsLinkedIn] = useState<boolean>(false);
+  const [linkedInNotionSidePanelStatus, setLinkedInNotionSidePanelStatus] = useState<boolean>(false);
+
+  chrome.runtime.onMessage.addListener((msg) => {
+    if (msg.name === 'linkedInNotionSidePanelStatus') {
+      setLinkedInNotionSidePanelStatus(msg.body.isOpen);
+    }
+  });
+
+  useEffect(() => {
+    chrome.tabs.query({ active: true, currentWindow: true }, async (tabs) => {
+      const url = tabs[0].url;
+      if (url.match(linkedInURLRegex)) {
+        setIsLinkedIn(true);
+        await chrome.tabs.sendMessage(tabs[0].id, 'toggleLinkedInNotionSidePanel'); // Message sent to the content and not to the background
+      }
+    });
+  }, []);
+
+  return (
+    <div
+      className={`plasmo-p-2 plasmo-w-${!isLinkedIn ? 52 : 40} plasmo-h-${
+        !isLinkedIn ? 32 : 20
+      } plasmo-flex plasmo-justify-center plasmo-items-center`}>
+      {!isLinkedIn ? (
+        <GoBackToLinkedInContent />
+      ) : (
+        <BaseParagraph className="plasmo-font-bold">
+          {linkedInNotionSidePanelStatus ? 'Enjoy 🎉' : 'Goodbye 👋'}
+        </BaseParagraph>
+      )}
+    </div>
+  );
```

Comment:
Here, instead of displaying the "Enjoy" and "Goodbye", I'd display **absolutely nothing**. It's uselessly distracting imo. Maybe something with a 'hidden' css class (to ensure that the popup js still executes).

Reply:
If we do so it displays an empty white square. I'll see what I can do 👌

Reply:
Impossible to make this square disappear and I find it very confusing...
@bvelitchkine Ok pour toi if we keep like this for now and we rediscuss the style and the wording during our styling taskforce that we'll probably do before first release?

Reply:
Yeah no problem, okfm